config/desktop/trixie/appgroups/browsers/armbian/create_desktop_package.sh

```
@@ -0,0 +1,4 @@
+# install optimized browser configurations
+cp "${SRC}"/packages/blobs/desktop/chromium.conf "${destination}"/etc/armbian
```

Shellcheck (0.10.0)

[warning] destination is referenced but not assigned.

(SC2154)